File: Res/Sorting/BubbleSort/bubbleSort.dart
List<int> bubbleSort(List<int> list) {
  int size = list.length;
  for (int i = 0; i < size; i++) {
    for (int j = 0; j < size - 1 - i; j++) {
      if (list[j] > list[j + 1]) {
        int temp = list[j];
        list[j] = list[j + 1];
        list[j + 1] = temp;
      }
    }
  }
  return list;
}

void main() {
  List<int> a = [5, 3, 2, 4, 1];
  List<int> sortedList = bubbleSort(a);
  print(sortedList);
}

File: Res/Sorting/BubbleSort/bubbleSort.py
def bubbleSort(list):
    size = len(list)
    for i in range(size):
        for j in range(size - i - 1):
            if list[j] > list[j + 1]:
                list[j], list[j + 1] = list[j + 1], list[j]
    return list

a = [85,52,36,12,21,45,87]
sortedList = bubbleSort(a)
print(sortedList)